src/Api/Controllers/AuthController.php

```diff
@@ -419,4 +420,24 @@ protected function sendEmail(BakaUsers $user, string $type) : void
     {
         return ;
     }
+
+    /**
+     * Validate auth token life status
+     *
+     * @return Response
+     */
+    public function isTokenAlive(): Response
+    {
+        $request = $this->request->getPostData();
+        $validation = new CanvasValidation();
+        $validation->add('access_token', new PresenceOf(['message' => _('access_token is required.')]));
```

Comment:
@rwhite27 shoudl they send us the token as a post or via the header? to validate?